main/zoe/api/zoe-contract-facet.md

```diff
@@ -166,23 +164,36 @@ Zoe.
 const { issuerKeywordRecord, keywords, terms } = zoe.getInstanceRecord()
 ```
 
-## zcf.getCurrentAllocation(offerHandle, sparseKeywords)
+## zcf.getCurrentAllocation(offerHandle, brandKeywordRecord)
 - `offerHandle` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
-- `sparseKeywords` sparseKeywords is an array of string keywords, which may be a subset of allKeywords.
-- Returns: <router-link to="/zoe/api/records.html#amount-keyword-record">`{<AmountKeywordRecord>}`</router-link>
+- `brandKeywordRecord` <router-link to="/zoe/api/records.html#brand-keyword-record">
+`{[<BrandKeywordRecord>]}`</router-link> is an optional record mapping keywords to brands for each of the offers.
```

Comment:
This method is about only one offer, so we shouldn't say "each of the offers" I think.